src/components/checkbox/Checkbox.tsx

```diff
@@ -13,6 +13,8 @@ export interface CheckboxProps {
     description?: React.ReactNode;
     /** fieldLabel - label for the field shown on top of the checkbox */
     fieldLabel?: React.ReactNode;
+    /** hideFocusStyles - whether the focus styles are hidden or not */
+    hideFocusStyles?: boolean;
```

Comment:
this looks like it might be a very specific use case. do we anticipate other apps would use this prop for checkboxes?

if this is for a specific component, would you be able to set the `border: none;` in that component's stylesheet?
```
&:focus + span::before {
    border: none;
}
```

Reply:
Yes, we can override the default styles from the component where this is consumer in EUA. It sounds like you think that's the best route, so I'll make that change---unless you say otherwise :)